.ci/scripts/dswx_hls/build_dswx_hls.sh

```diff
@@ -32,7 +32,7 @@ echo "TAG: $TAG"
 echo "SAS_IMAGE: $SAS_IMAGE"
 
 # Check that the .ci scripts directory exists
-if [ ! -d "${WORKSPACE}/.ci" ]; then
+if [ ! -d "${WORKSPACE}" ]; then
```

Comment:
The other build scripts have this same check on `${WORKSPACE}/.ci`, so why was it only changed for DSWx-HLS?

Reply:
Fixed